static/js/Misc.jsx

```diff
@@ -23,6 +23,9 @@ import Cookies from "js-cookie";
 import {EditTextInfo} from "./BookPage";
 import ReactMarkdown from 'react-markdown';
 import TrackG4 from "./sefaria/trackG4";
+import {trackSidebarAdClick, trackSidebarAdImpression} from "./Promotions";
+const cookie = Sefaria._inBrowser ? $.cookie : Sefaria.util.cookie;
```

Comment:
What are the product requirements, only locally to hide the button after first click? or long term for logged in users